src/components/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as SearchBar } from "./SearchBar";
```

Comment:
I hadn't seen this before. Is this a best practice you saw somewhere?

Reply:
I'm not sure about best practice, but we did it at my last company, and I like how it makes for cleaner imports.